types/nock/index.d.ts

```diff
@@ -86,14 +86,15 @@ declare namespace nock {
     }
 
     export interface Interceptor {
-        query(params: boolean | { (querObject: any): boolean; } | any): this;
+        query(params: boolean | { (queryObject: any): boolean; } | any): this;
 
         reply(responseCode: number, body?: string | any, headers?: HttpHeaders): Scope;
         reply(responseCode: number, callback: (uri: string, body: string, cb?: ReplyCallback) => ReplyCallbackResult, headers?: HttpHeaders): Scope;
         reply(callback: (uri: string, body: string, cb?: ReplyCallback) => ReplyCallbackResult, headers?: HttpHeaders): Scope;
         replyWithError(errorMessage: string | any): Scope;
         replyWithFile(responseCode: number, fileName: string, headers?: HttpHeaders): Scope;
 
+        matchHeader(name: string, value: string | RegExp | { (value: string): boolean; }): this;
```

Comment:
Any reason you're using a callable object type here instead of a function type?

Reply:
I copied the line from the other instance above, but I'd be happen to update both cases if that's a prefered syntax.
```ts
matchHeader(name: string, value: string | RegExp | ((value: string) => boolean)): this;
```

Reply:
No, I'm for consistency here.